### PR TITLE
Made styles more specific

### PR DESCRIPTION
**Pull request by obgibson**
Screen Shot 2021-12-14 at 5 35 39 PM

to 

Screen Shot 2021-12-14 at 4 47 58 PM

The change was to make all the existing styles more specific by namespacing them by `.card`

### CODE REVIEW

**Comment by obgibson** [comment]
Note that the DAG still does not show the connectors.

File: metaflow/plugins/cards/ui/src/global.css [hunk 1]
```diff
@@ -31,119 +31,15 @@
...
-  background: var(--bg);
-  color: var(--black);
-  font-family: "Roboto", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto",
-    "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  font-size: 14px;
-  font-weight: 400;
-  line-height: 1.5;
-  text-size-adjust: 100%;
+html, body {
```
Comment (seethroughdev) [inline metaflow/plugins/cards/ui/src/global.css:34]: why not just apply these to `.card` as well and remove global.css?

File: metaflow/plugins/cards/ui/src/App.svelte [hunk 2]
```diff
@@ -12,7 +13,7 @@
   import Nav from "./components/aside-nav.svelte";
 </script>
 
-<div class="container">
+<div class="container card">
```
Comment (seethroughdev) [inline metaflow/plugins/cards/ui/src/App.svelte]: do we risk `card` being too generic if we don't control the whole page?  Its a pretty common UI term.  I wonder if instead we use an ID to to establish precedence over any classes?  ie.  find/replace `.card` with `#app-card`

File: metaflow/plugins/cards/ui/src/App.svelte [hunk 3]
```diff
@@ -3,6 +3,7 @@
   import "./prism";
   import "./global.css";
```
Comment (seethroughdev) [inline metaflow/plugins/cards/ui/src/App.svelte:4]: do we need this at all now?

**Comment by seethroughdev** [comment]
overall looks fine, I thought we were going to have to address every `:global` reference too.  I suppose its not necessary if it still adds the scoped classes inside global?  it would be helpful to verify though.

my main concern is that we're using a class and its a common name (card).  wondering if it makes more sense to create an ID to namespace instead.  ie.  `#mf-card` or something.

**Review by seethroughdev — CHANGES_REQUESTED** [review]

File: metaflow/plugins/cards/ui/src/global.css (hunk 1 above)
Comment (obgibson) [reply metaflow/plugins/cards/ui/src/global.css:34]: These remove extra padding around the `.card` when it is within our page. The same rules are in `.card` too for when we are embedded.

File: metaflow/plugins/cards/ui/src/App.svelte (hunk 3 above)
Comment (obgibson) [reply metaflow/plugins/cards/ui/src/App.svelte:4]: This is for rules at the index.html level, which we still have. I left the variables in there too

File: metaflow/plugins/cards/ui/src/App.svelte (hunk 2 above)
Comment (obgibson) [reply metaflow/plugins/cards/ui/src/App.svelte]: I agree we should change the name to something less generic - I like `mf-card`. I considered an ID but thought there may be multiple of these cards on a page. But your point about increased specificity may override that

Comment (seethroughdev) [reply metaflow/plugins/cards/ui/src/App.svelte]: good point, lets go with the more specific class then.

**Review by seethroughdev — APPROVED** [review]
assuming you change the class, looks good to me!

**Comment by obgibson** [comment]
on it

**Pull request merged**